spring-boot-project/spring-boot-actuator-autoconfigure/src/docs/asciidoc/endpoints/configprops.adoc

```diff
@@ -26,3 +26,26 @@ The following table describes the structure of the response:
 
 [cols="2,1,3"]
 include::{snippets}/configprops/response-fields.adoc[]
+
+
+[[configprops-retrieving-by-prefix]]
```

Comment:
@wilkinsona I absolutely ❤️ what you did here w/ the endpoint doc generation - clever. It provides a consistent and accurate set of docs as well as provides some test coverage. Thank you.

Reply:
That's lovely to hear. Thank you!